contrib/mmap/src/MapBuilder.h

```diff
@@ -121,8 +127,10 @@ namespace MMAP
             bool m_skipContinents;
             bool m_skipJunkMaps;
             bool m_skipBattlegrounds;
-            bool m_quick;
+            bool m_quick;       //option set via commandline parameter
```

Comment:
Remember to always initialize any new member variables you add in the constructor of the class.